cloudinit/cmd/devel/hotplug_hook.py

```diff
@@ -237,6 +242,22 @@ def handle_hotplug(hotplug_init: Init, devpath, subsystem, udevaction):
         raise last_exception
 
 
+def enable_hotplug(hotplug_init: Init, subsystem):
+    LOG.debug("Fetching datasource")
+    datasource = hotplug_init.fetch(existing="trust")
+    if not datasource:
+        return
+    if not datasource.get_supported_events([EventType.HOTPLUG]):
```

Comment:
We aren't doing any validation of specific subsystem scope at network here versus some other hotplug. We are only asking the datasource if any hotplug is supported. But, maybe this is ok since our CLI limits the subsystem scope to `-s net` anyway and we don't have any other scope definitions besides EventScope.NETWORK

Reply:
Added verification of HOTPLUG enabled in the selected scope.